src/generate/formats/writer/yaml/utils.ts

```diff
@@ -369,8 +369,8 @@ export async function extractComponentList(
   const { flows, requiredConfigVars } = integration;
 
   // Flows
-  flows.forEach((flow) => {
-    flow.steps.forEach((step) => {
+  (flows ?? []).forEach((flow) => {
+    (flow.steps ?? []).forEach((step) => {
```

Comment:
Both these states should be impossible, right? All integrations must have at least one flow and each must have at least a trigger?

Reply:
Yeah admittedly I was full of fear of more errors but realistically you're correct. I'll remove these checks